3dhubs_amsterdam_python_flask.html

```diff
@@ -0,0 +1,64 @@
+---
+title:  Python Developer
+company: 3D Hubs
+url: https://www.3dhubs.com/
+location: Amsterdam, the Netherlands
```

Comment:
You can be more specific - for example, you can include a postal code. This may be useful to candidates who want to work in a specific location.